constants.py

```diff
@@ -1,56 +1,67 @@
 import discord
+from config import settings
 
 HELP_REPLY = discord.Embed(
     title="Ferris Elf help page",
     color=0xE84611,
-    description="""
+    description=f"""
 **help** - Send this message
 **info** - Some useful information about benchmarking
 **best _[day]_ _[part]_** - Best times so far for a day
 **submit _[day]_ _[part]_ <attachment>** - Benchmark attached code
 
 If [_day_] and/or [_part_] is ommited, they are assumed to be today and part 1
 
-Message <@{}> for any questions""",
+{settings.discord.support_info}""",
 )
 
 INFO_REPLY = discord.Embed(
     title="Benchmark information",
     color=0xE84611,
-    description="""
-When sending code for a benchmark, you should make sure it looks like.
+    description=f"""**Submissions**
+When sending code for a benchmark, you should make sure it looks like this:
+
 ```rs
-pub fn run(input: &str) -> i64 {
+pub fn run(input: &str) -> i64 {{
     0
-}
+}}
 ```
 
 Input can be either a &str or a &[u8], which ever you prefer. The return should \
-be the solution to the day and part. 
+be the solution to the day and part.
+
+Rust version is {{settings.discord.rust_version_info}}.
 
-Rust version is latest Docker nightly
 **Available dependencies**
+
 ```toml
 bytemuck = "1"
 itertools = "0.10"
 rayon = "1"
 regex = "1"
 parse-display = "0.6"
 memchr = "2"
-core_simd = { git = "https://github.com/rust-lang/portable-simd" }
 arrayvec = "0.7"
 smallvec = "1"
 rustc-hash = "1"
 bitvec = "1"
 dashmap = "5"
-atoi_radix10 = { git = "https://github.com/gilescope/atoi_radix10" }
 btoi = "0.4"
+nom = "7"
+ascii = "1.1.0"
 ```
```

Comment:
Would it make sense to read this directly from the runner's Cargo.toml?

Reply:
It absolutely would, but I'm going to punt that to [a future issue](https://github.com/proegssilb/ferris-elf/issues/58) . Not saying it's hard, just out-of-scope for this issue + change.